packages/inter-protocol/scripts/add-collateral-core.js

```diff
@@ -70,6 +70,18 @@ export const psmGovernanceBuilder = async ({
           psm: publishRef(
             install('../src/psm/psm.js', '../bundles/bundle-psm.js'),
           ),
+          vaults: publishRef(
+            install(
+              '../src/vaultFactory/vaultFactory.js',
+              '../bundles/bundle-vaults.js',
```

Comment:
this is bundled elsewhere as,
```suggestion
              '../bundles/bundle-vaultFactory.js',
```

Reply:
done